crates/adapters/celestia/Cargo.toml

```diff
@@ -20,8 +20,8 @@ bech32 = { workspace = true }
 prost = "0.13"
 
 # Upgrade with care. New version might be incompatible
```

Comment:
Can we remove this comment now and unpin?

Reply:
I think we can unpin patch version, but still need to stick to minor version and upgrade with care. It is still pre 1.0, need to check up with Eiger if they plan any breaking changes soon.